spec/spec_helper.rb

```diff
@@ -94,3 +95,15 @@
   Kernel.srand config.seed
 =end
 end
+
+def stub_food_request
```

Comment:
Love this helper method. However, it doesn't look like you ever utilize VCR, because you did not add the configuration block, and you never use the vcr syntax in any of your tests to use the cassette.